Point_set_shape_detection_3/include/CGAL/Shape_detection_3/Region_growing.h

```diff
@@ -34,8 +34,10 @@
 #include <CGAL/squared_distance_3.h>
 #include <CGAL/function.h>
 
-#include <CGAL/boost/iterator/counting_iterator.hpp>
-#include <CGAL/boost/iterator/transform_iterator.hpp>
+#include <boost/iterator/counting_iterator.hpp>
+#include <boost/iterator/transform_iterator.hpp>
```

Comment:
That patch is probably not right. @sgiraudot, can you comment it? I think the new headers `<CGAL/boost/...>` are used to remove warnings...

Reply:
Looks I made a mistake on a conflict resolution.